quipucords/client/index.html

```diff
@@ -1,84 +1 @@
-<!DOCTYPE html>
```

Comment:
This looks like an accidental left over bit?

Reply:
yup, thanks